spec/controllers/openid_connect/token_controller_spec.rb

```diff
@@ -79,13 +79,4 @@
       end
     end
   end
-
-  describe '#options' do
-    it 'is empty so it can be used as a pre-flight request' do
-      process :options, 'OPTIONS'
```

Comment:
does 5.1 no longer support sending OPTIONS requests? I really like having this spec because it makes sure we handle the OPTIONS which is important for CORS